app/src/main/java/com/blackboxteam/app/blackbox/LoginScreenActivity.java

```diff
@@ -1,13 +1,79 @@
 package com.blackboxteam.app.blackbox;
 
-import android.support.v7.app.AppCompatActivity;
+import android.content.Intent;
 import android.os.Bundle;
+import android.support.design.widget.TextInputLayout;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import com.blackboxteam.app.blackbox.util.Validation;
+
+import static com.blackboxteam.app.blackbox.util.Validation.validateEmail;
+import static com.blackboxteam.app.blackbox.util.Validation.validateMinLength;
+
+public class LoginScreenActivity extends AppCompatActivity implements View.OnClickListener {
 
-public class LoginScreenActivity extends AppCompatActivity {
+    Button loginBtn;
+    EditText etEmail, etPassword;
+    TextInputLayout tiEmail, tiPassword;
+    Validation validate;
```

Comment:
All member variables should be declared private and in separate lines.

Reply:
Also, the ideal naming convention for member variable is: m<VariableName>. Where VariableName is a concise description of the variable. e.g 
`private GoogleAPIClient mGoogleApiClient;`
`private String mUserEmail;`
`private TextView mUserEmailTextView;`